CryptoBinPkg/Driver/Packaging/generate_cryptodriver.py

```diff
@@ -920,7 +932,7 @@ def get_supported_library_types(phase):
         return ["DXE_RUNTIME_DRIVER"]
     elif phase == "SMM":
         return ["DXE_SMM_DRIVER", "SMM_CORE"]
-    elif phase == "STANDALONEMM":
+    elif phase == "STANDALONEMM" or phase == "STANDALONEMM_MMSUPV":

```

Comment:
optional, feel free to ignore.
```suggestion
    elif phase in {"STANDALONEMM","STANDALONEMM_MMSUPV"}:

```